hello-world-python/gradle.properties

```diff
@@ -0,0 +1,4 @@
+#
+#Thu Jan 06 17:15:36 GMT 2022
+conclaveVersion=1.2.1
+conclaveRepo=../repo
```

Comment:
I am not even sure if we should set the ConclaveVersion. Otherwise, the samples we have out dated code every time Conclave is released. I will leave the decision to @shamsasari.

Reply:
yes, you are correct I have updated the same.